PugSharp.Match/Match.cs

```diff
@@ -209,7 +209,7 @@ private void ScrambleTeams()
 
     private void ContinueIfDefault()
     {
-        if (MatchInfo.Config.TeamMode == Config.TeamMode.Default)
+        if (MatchInfo.Config.TeamMode == Config.TeamMode.Default || MatchInfo.Config.TeamMode == Config.TeamMode.PlayerSelect)
```

Comment:
Either Change the Method name or create a second method `ContinueIfPlayerSelect`

Reply:
Added new method in latest commit